meta-oe/recipes-devtools/exprtk/exprtk_git.bb

```diff
@@ -0,0 +1,22 @@
+SUMMARY = "Expression parser"
+HOMEPAGE = "https://github.com/ArashPartow/exprtk"
+SECTION = "libs"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+SRC_URI = "git://github.com/ArashPartow/exprtk.git;nobranch=1"
```

Comment:
why is nobranch needed ?

Reply:
Wanted to take master branch only, I will remove this.